Exiled.API/Features/Items/Scp330.cs

```diff
@@ -71,6 +73,34 @@ public bool AddCandy(CandyKindID type)
             return false;
         }
 
+        /// <summary>
+        /// Adds a collection of candy's to a bag.
+        /// </summary>
+        /// <param name="candies">The <see cref="CandyKindID"/>'s to add.</param>
+        /// <returns> <see cref="CandyAddStatus"/> based on insertion status. </returns>
+        public CandyAddStatus AddCandy(IReadOnlyCollection<CandyKindID> candies)
```

Comment:
IEnumerable maybe?

Reply:
> 

Trying to avoid expensive reflection; however, I understand. Sounds good, hope to hear from you soon (I got 7 hours of sleep, we have made a little progress)